packages/keystone/tracing.js

```diff
@@ -0,0 +1,56 @@
+const opentelemetry = require('@opentelemetry/api')
+const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-proto')
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-proto')
+const { PgInstrumentation } = require('@opentelemetry/instrumentation-pg')
+const { PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics')
+const opentelemetrySDK = require('@opentelemetry/sdk-node')
+
+const KeystoneInstrumentation = require('./KeystoneInstrumentation')
+
+const init = ({ tracesUrl = 'http://localhost:4318/v1/traces', metricsUrl = 'http://localhost:4318/v1/metrics' } = {}) => {
```

Comment:
Done, you now can provide the app with

OTEL_CONFIG env variable, which can include these fields:

{ enabled, tracesUrl, metricsUrl, headers = {} }

Auth token should be supported, since it is only a header